src/main/java/transforms/GeocoderTransform.java

```diff
@@ -68,7 +68,19 @@ public String getAddress( String input, String addressObject ) {
         if ( outercodes.contains( addressObject ) ) {
             return (String) address.get( addressObject );
         } else if ( addressObject.equals("geographypoint") ) {
-            return address.get( "lat" ) + "," + address.get("lon");
+            String latitude = address.get( "lat" ).toString();
+            String longitude = address.get("lon").toString();
+
+            String lat_suffix = "", lon_suffix = "";
+            if (!latitude.contains(".")) {
+                lat_suffix = ".0";
+            }
+
+            if (!longitude.contains(".")) {
+                lon_suffix = ".0";
+            }
+            return new StringBuilder(latitude).append(lat_suffix).append(",")
+                    .append(longitude).append(lon_suffix).toString();
```

Comment:
Could also rewrite this section starting with the `StringBuilder(latitude)` at the top and then `if (contains) { append(".0") } else { append("") }` for the conditionals and returning `.toString()`